UnitTests/SecurityVulnerabilitiesTest.cs

```diff
@@ -71,6 +71,10 @@ public void InvalidCurveAttack()
                 JWT.Decode(attackMod113, privateKey);
                 Assert.True(false, "Should fail with CrytographicException");
             }
+            catch (PlatformNotSupportedException e)
```

Comment:
Probably good idea to have unix version of test, just be safe it's not vulnerable to same attack. Can add myself.

Reply:
Added a theory to test the ECDH key implementation as well.